rules-topics/01_introduction.md

```diff
@@ -62,6 +62,56 @@ Welcome to Chronicles. Whether you've been LARPing for years, or have never seen
 tabard in your life, Chronicles welcomes you. For the newer players, here are a few tips
 to get you started
 
+### Creating a Character
+
+If you are a new player you will start with 100 Skill Points 10 Body and 5 Armor Training.  Skill Points are used to purchase skills like the ability to swing a sword or cast a spell.  You can use the Character Calculator to design your character.  
```

Comment:
Suggestion here:`All new players start with 100 Skill Points, 10 body, and 5 armor.  The 5 armor is considered a free bonus as long as the player is wearing garb or a costume.`
We were missing commas, needed to lowercase "Body", and update "Armor Training" to just armor so that it doesn't potentially read like we give them 5 purchases of the skill.

Reply:
Updated